main.go

```diff
@@ -174,6 +159,14 @@ func main() {
 		log.Infof("Host: %s", *nutanixURL)
 		nutanixAPI := nutanix.NewNutanix(*nutanixURL, *nutanixUser, *nutanixPassword, maxParallelReq)
 
+		// Start health ticker for this section (use collection interval if available)
+		collectionInterval := 30 // default
+		if ok && conf.MaxParallelRequests > 0 {
+			// Use a reasonable interval based on parallel requests
+			collectionInterval = 30 // could be made configurable
+		}
+		nutanix.StartHealthTicker(stopTicker, collectionInterval)
```

Comment:
How do you ensure that this gets called only once per configuration? Seems to me like this is started every time the API request come in, no?